src/object.d

```diff
@@ -1526,41 +1526,73 @@ const:
 
     @property uint flags() nothrow pure @nogc { return _flags; }
 
+    /************************
+     * Returns:
+     *  module constructor for thread locals, `null` if there isn't one
+     */
     @property void function() tlsctor() nothrow pure @nogc
     {
         return flags & MItlsctor ? *cast(typeof(return)*)addrOf(MItlsctor) : null;
     }
 
+    /************************
+     * Returns:
+     *  module destructor for thread locals, `null` if there isn't one
+     */
     @property void function() tlsdtor() nothrow pure @nogc
     {
         return flags & MItlsdtor ? *cast(typeof(return)*)addrOf(MItlsdtor) : null;
     }
 
+    /*****************************
+     * Returns:
+     *  address of a module's `const(MemberInfo)[] getMembers(string)` function, `null` if there isn't one
+     */
     @property void* xgetMembers() nothrow pure @nogc
     {
         return flags & MIxgetMembers ? *cast(typeof(return)*)addrOf(MIxgetMembers) : null;
     }
 
+    /************************
+     * Returns:
+     *  module constructor, `null` if there isn't one
+     */
     @property void function() ctor() nothrow pure @nogc
     {
         return flags & MIctor ? *cast(typeof(return)*)addrOf(MIctor) : null;
     }
 
+    /************************
+     * Returns:
+     *  module destructor, `null` if there isn't one
+     */
     @property void function() dtor() nothrow pure @nogc
     {
         return flags & MIdtor ? *cast(typeof(return)*)addrOf(MIdtor) : null;
     }
 
+    /************************
+     * Returns:
+     *  module order independent constructor, `null` if there isn't one
+     */
     @property void function() ictor() nothrow pure @nogc
     {
         return flags & MIictor ? *cast(typeof(return)*)addrOf(MIictor) : null;
     }
 
+    /*************
+     * Returns:
+     *  address of function that runs the module's unittests, `null` if there isn't one
+     */
     @property void function() unitTest() nothrow pure @nogc
     {
         return flags & MIunitTest ? *cast(typeof(return)*)addrOf(MIunitTest) : null;
     }
 
+    /****************
+     * Returns:
+     *  array of pointers to the ModuleInfo's of modules imported by this one
```

Comment:
You didn't mention that this function can return null.

Again, brevity hurts clarity in this case. You should probably start the sentence with "_an_ array of pointers".

Please add a particle to specify which "modules" are being referred to ("_the_ modules", "_all_ modules", "_any_ module", etc.).

Also recommended: "...modules imported by this one" -> "...modules imported by this module". Clarity, explicitness and uniformity are more valuable than brevity when it comes to documentation.

Finally, "ModuleInfo's" is bad style and should be written as ModuleInfos in the plural case (you can write it as `ModuleInfo`s to make it clear that the 's' is only for marking plurality and not part of the symbol name).